xmlseclibs.php

```diff
@@ -374,7 +374,6 @@ public static function getRawThumbprint($cert) {
                 }
             } else {
                 if (strncmp($curData, '-----END CERTIFICATE', 20) == 0) {
-                    $inData = FALSE;
```

Comment:
That's what I thought in the beginning. However, if you look at it closely, the next line is a break statement, exiting the loop. The `$inData` variable is only used inside the loop in this method, so as soon as we are exiting the loop it is no longer used, and its value is therefore irrelevant.